scripts/deploy_assisted_service.sh

```diff
@@ -38,6 +38,9 @@ if [[ "${ENABLE_KUBE_API}" == "true" || "${DEPLOY_TARGET}" == "operator" ]]; the
     if [ -z "${RELEASE_IMAGES:-}" ]; then
         export RELEASE_IMAGES='[]'
     fi
+    # The kube-api flow sets all infra env image types to the ISO image type configured
+    # when the service is deployed
+    export ISO_IMAGE_TYPE=${ISO_IMAGE_TYPE:-minimal-iso}
```

Comment:
Needed to include this since infra envs of kube api flow will inherit the assisted service's deployment's value for ISO_IMAGE_TYPE

More details in comment: https://issues.redhat.com/browse/MGMT-11566?focusedCommentId=21019844&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-21019844

Reply:
Is that a bug - `ISO image type can't be specified per infra-env in kube api deployments of assisted-service` ?

Reply:
Not sure if it was intentional - I've asked the slack channel to see if anyone knows! If it's a bug I'll open up a defect and make note to change it in this code once it's fixed.

Reply:
OK, if it's not temporary fix I'm OK with it.
Let's wait for QE approval, but note that it might take a while due to the holiday shutdown.

Reply:
Ok sounds good! Not a problem since this is not urgent. Thanks!